### DISCUSSION TITLE
Published content and CRs are correct, but regular GitBook view does not reflect the changes

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#831] [0. General bugs and questions]
For one of my spaces, the published content is correct, and when I create a CR the content that I edit is correct, but in non-editing GitBook view the content does not reflect the latest changes.

For example, on the home page, I have a couple of sets of cards. The published version (correct) looks like:

![image](https://github.com/user-attachments/assets/a7e82b99-dd9a-4061-a108-3c0bd858c8b2)

In GitBook, it seems like it is in an intermediate state, where I was in the process of splitting the single set of cards into two.

![image](https://github.com/user-attachments/assets/303dbf70-61b0-4673-8e30-257a186f4d06)

I just made some changes this morning, which involved moving and replacing the content of a topic. The published content shows the new content.

![image](https://github.com/user-attachments/assets/471b218a-e972-4778-b2e9-347a2e7868b4)

GitBook, however, reflects the move and the new title, but not the new content.

![image](https://github.com/user-attachments/assets/b4c742c0-9086-4901-8f33-d00933904c15)

I've tried reloading, clearing the browser cache, and opening GitBook in an incognito window, but it doesn't fix it.

**Comment by dbasilfid** [comment]
I am seeing the same issue with GitBook UI not accurately rendering content changes although they do show up correctly on the live site.

**Comment by addisonschultz** [comment]
Hi @JaniceManwiller @dbasilfid - is this issue still standing? I’m wondering if there’s any caching going on

**Reply by JaniceManwiller** [reply]
I am still seeing it. When I first saw it, I did try clearing the browser history, but that didn't fix it.

**Reply by JaniceManwiller** [reply]
I just merged a CR that edited the second page that I highlighted in the original post. That page is now displaying correctly in the regular GitBook view.

The other page is still incorrect.

Also, the regular GitBook view indicates that there are broken links from the incorrect page (displays a broken link count at the top), which reflects the state of the page as it is displayed but not the actual content, which is fine.

**Reply by JaniceManwiller** [reply]
I am also seeing the same issue when I try a different browser.

Also, not sure if this is related, but for the doc site that contains the space, the search is not working. When I click the search field, I don't see the usual list of suggested questions, and a search returns the "something went wrong" error.

**Reply by addisonschultz** [reply]
@JaniceManwiller can you let us know what Change Requests (Link to it) and pages that are out of date? We’ve been looking into this the last day and any info you can send us would be very helpful. 

You can send it to me at addison @ gitbook.com

**Comment by JaniceManwiller** [comment]
So this is a little tricky, because I've made other changes since then to
force some of the pages to refresh.

But here are a few examples.


For the Textual doc space, this is the first change request (that I
noticed) for which the changes did not correctly display in GitBook -
https://app.gitbook.com/o/-LSQCLFPRklDIrR0HOsF/s/vOPn7KQptPWmS5iKg5P0/~/changes/161/tonic-textual-api/textual-api-instantiate-sdk/~/overview

After this CR, the home page was not displaying correctly in GitBook -
https://app.gitbook.com/o/-LSQCLFPRklDIrR0HOsF/s/vOPn7KQptPWmS5iKg5P0/ -
However, since then I merged a CR that made small edits to the page, and it
now displays correctly, although I am still seeing the incorrect broken
links icon.

This other CR -
https://app.gitbook.com/o/-LSQCLFPRklDIrR0HOsF/s/vOPn7KQptPWmS5iKg5P0/~/changes/162/textual-playground/~/overview
included moving and changing the content of the linked topic. The content
did not display correctly until after I made another change to it.

Here is an example of a page that is still incorrect in GitBook but correct
in the published content and CRs:
https://app.gitbook.com/o/-LSQCLFPRklDIrR0HOsF/s/vOPn7KQptPWmS5iKg5P0/~/diff/pipelines/pipeline-setup/pipeline-create-edit-delete
- it includes multiple options to create a pipeline. There is now only one
option.

The published page is
https://docs.tonic.ai/textual/pipelines/pipeline-setup/pipeline-create-edit-delete


In the Structural doc space, this change request added a new topic that was
also added as a card on the parent page -
https://app.gitbook.com/o/-LSQCLFPRklDIrR0HOsF/s/-LSQCLFQ4bslJ-HYc8c3/~/changes/1456/workflows/data-generation-run-job/~/overview

The new card does not display in GitBook -
https://app.gitbook.com/o/-LSQCLFPRklDIrR0HOsF/s/-LSQCLFQ4bslJ-HYc8c3/workflows/data-generation-run-job

It does display in the published content -
https://docs.tonic.ai/app/workflows/data-generation-run-job


It also looks like changes in other CRs that I merged Christmas and New
Year's weeks are not reflected in the GitBook display.  CRs from this week
seem to reset the pages that they change, but other incorrect pages remain
incorrect.

Hope that helps.

**Reply by addisonschultz** [reply]
Thanks @JaniceManwiller, we’re looking into this

**Reply by JaniceManwiller** [reply]
Any update on this? I am still seeing the incorrect content in the regular GitBook view, as well as the incorrect broken links indicator for one of my spaces.

Just another note that I also see the incorrect content when I log in on different machines, so if it is a caching issue it's not just the browser cache.

**Reply by taranvohra** [reply]
Hi @JaniceManwiller, can you give us links of pages which you believe are showing incorrect content in regular GitBook Space view? Is it the same one from above? We've made some fixes over the last week which should not cause this problem to occur again but I'd still like to re-initialize the pages you feel are outdated.

**Reply by JaniceManwiller** [reply]
This might not be comprehensive, but here are the examples that I know of right now - there was another one yesterday, but that seems to be resolved now:

https://app.gitbook.com/o/-LSQCLFPRklDIrR0HOsF/s/-LSQCLFQ4bslJ-HYc8c3/workflows/data-generation-run-job - Missing a card.

https://app.gitbook.com/o/-LSQCLFPRklDIrR0HOsF/s/vOPn7KQptPWmS5iKg5P0/ - The general content is correct, but GitBook is claiming that there are two broken links here, which there aren't. I moved the target topics for a couple of the cards, but the links are updated and work correctly.

**Reply by taranvohra** [reply]
Thanks for providing the details. I've re-initialized the page at https://app.gitbook.com/o/-LSQCLFPRklDIrR0HOsF/s/-LSQCLFQ4bslJ-HYc8c3/workflows/data-generation-run-job and it seems to be showing the latest content. Please let us know if it comes up again

As for the broken links, we'll investigate it separately

**Reply by JaniceManwiller** [reply]
Uncovered another one - as I mentioned earlier, I had done general cleanup CRs (typos, missing links, terminology fixes) that touched a lot of topics, and unfortunately this coincided with the issue of pages not being updated in regular GitBook view.

In GitBook -  https://app.gitbook.com/o/-LSQCLFPRklDIrR0HOsF/s/-LSQCLFQ4bslJ-HYc8c3/admin/tonic-user-access/permissions/permission-sets-builtin

![image](https://github.com/user-attachments/assets/2d02a0ba-5ec6-4f5c-9eca-80a9c57880f3)

In the published content:

![image](https://github.com/user-attachments/assets/1b423fc9-d6b8-4a53-b529-1e0c2da97d9c)
